### PR TITLE
fix annoying horizontal scroll bar in repos in dabshboard

### PR DESCRIPTION
**Pull request by forrestbao**
For issue  https://github.com/codepod-io/codepod/issues/118

### CODE REVIEW

File: ui/src/pages/repos/CreateRepoForm.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ interface form {
   button?: any;
 }
 
-export default function CreateRepoForm(props: form = {}) {
```
Comment (lihebi) [inline ui/src/pages/repos/CreateRepoForm.tsx:24]: The interface form is unused now. Remove it?

**Review by lihebi — COMMENTED** [review]
You didn't seem to have used a formatted. If you are using VSCode, try to format these two files using [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).

**Comment by lihebi** [comment]
Can you also give me permission to push to your branch, so that I can make these trivial tweaks as well?

File: ui/src/pages/repos/CreateRepoForm.tsx (hunk 1 above)
Comment (forrestbao) [reply ui/src/pages/repos/CreateRepoForm.tsx:24]: Yeah, in the new line, I removed the type `form` and default value `{}` but left the `props` argument there for easy customization and control in the future. Good?

**Comment by forrestbao** [comment]
> Can you also give me permission to push to your branch, so that I can make these trivial tweaks as well?

Of course, invitation sent

File: ui/src/pages/repos/CreateRepoForm.tsx (hunk 1 above)
Comment (lihebi) [reply ui/src/pages/repos/CreateRepoForm.tsx:24]: Sounds good.

**Pull request closed without merging**